src/CodeStyle/CSharp/Tests/Microsoft.CodeAnalysis.CSharp.CodeStyle.UnitTests.csproj

```diff
@@ -4,19 +4,55 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.CSharp</RootNamespace>
+    <DefineConstants>$(DefineConstants),CODE_STYLE</DefineConstants>
     <TargetFramework>net472</TargetFramework>
+    <IsShipping>false</IsShipping>
```

Comment:
😕 How is this not the default for test projects?

Reply:
I am not sure either. But I started seeing some CA violations in these test projects, for rules which were already turned of in our NonShipping ruleset.

Reply:
It is the default: https://github.com/dotnet/arcade/blob/671bfe2954fe2e0fc2052cb0ee59cbfe9e5e8801/src/Microsoft.DotNet.Arcade.Sdk/tools/Tests.props#L37

Rather than setting it to false here, we should investigate why this doesn't work.

Reply:
My bad here - I needed to define this for a test utility project, and went ahead adding it for the C# and VB CodeStyle unit test projects simultaneously. I will revert this.